lib/compiler/build_runner.zig

```diff
@@ -1364,6 +1373,10 @@ fn usage(b: *std.Build, out_stream: anytype) !void {
         \\  --zig-lib-dir [arg]          Override path to Zig lib directory
         \\  --build-runner [file]        Override path to build runner
         \\  --seed [integer]             For shuffling dependency traversal order (default: random)
+        \\  --build-id[=style]           At a minor link-time expense, coordinates stripped binaries
```

Comment:
Copy/paste error?

Reply:
same text as used in main
https://github.com/ziglang/zig/blob/5b4759bd3c860b0648bf847ba9aa52584a9547d9/src/main.zig#L582

I think the goal here is to imply the ability to ensure debug info came from a specific binary, though I agree that its worth rewriting.